internal/persistence/mysql/mysql.go

```diff
@@ -16,7 +17,11 @@ import (
 )
 
 // ErrMariaDBUnsupported is the error returned when db.DialectMariaDB is attempted to be used.
-var ErrMariaDBUnsupported = errors.New("MariaDB is currently unsupported")
+var (
+	ErrMariaDBUnsupported = errors.New("MariaDB is currently unsupported")
+
+	duplicateEntryErrorCode uint16 = 0x426
```

Comment:
Why not just write out `duplicateEntryErrorCode uint16 = 1062`? Any reason to specifically use hex?

Reply:
not really. that what I had when debugging... will use 1062 for clarity with the mysql docs.